airbyte-integrations/connectors/source-shopify-singer/Dockerfile

```
@@ -10,7 +10,7 @@ ENV AIRBYTE_IMPL_PATH="SourceShopifySinger"
 WORKDIR /airbyte/integration_code
 COPY $CODE_PATH ./$CODE_PATH
 COPY setup.py ./
-RUN pip install tap-shopify==1.2.6
+RUN pip install https://github.com/ns-admetrics/tap-shopify/archive/bump-shopify-api-version.zip
```

@ns-admetrics could you instead use the tap version we forked: 
```suggestion
RUN pip install https://github.com/airbytehq/tap-shopify/tarball/master
```